ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -249,6 +273,8 @@ define([
                     function(_this) { _this.clearUtmLowerRight(true); });
 
                 this.set(result);
+            } else if (this.get('locationType') === "dms") {
+                this.setBboxDmsFromMap();
```

Comment:
❓ Does this need to call setRadiusDmsFromMap?

Reply:
No, because `setLatLon()` is only called from [here](https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js#L73), which is only triggered by changes to the bounding box coordinates.